packages/test/test-end-to-end-tests/src/test/container.spec.ts

```diff
@@ -60,6 +60,7 @@ const timeoutMs = 500;
 describeNoCompat("Container", (getTestObjectProvider) => {
 	let provider: ITestObjectProvider;
 	const loaderContainerTracker = new LoaderContainerTracker();
+	const mockLogger = new MockLogger();
```

Comment:
won't this swallow all logs generated by these test and prevent them from reaching telemetry?

Reply:
Oh geez

Reply:
thats not a thing

Reply:
<img width="445" alt="image" src="https://github.com/microsoft/FluidFramework/assets/12305068/3ee3443f-c0c9-4f39-a535-b4091a64da20">
?